d2bs/kolbot/libs/common/Config.js

```diff
@@ -415,7 +415,10 @@ var Config = {
 	BattleOrders: {
 		Mode: 0,
 		Getters: [],
-		Wait: false
+		Wait: false,
+		QuitOnFailedGive: false,
+		QuitOnFailedGet: false,
```

Comment:
The QuitOnFailed[Give|Get] might can be the same value, as you can either give or get, never both.

Reply:
oh yep, you're right. Could be reused